install_requirements.sh

```diff
@@ -85,7 +85,7 @@ DEVEL_REQUIREMENTS=(
 # TODO(dbort): Make each example publish its own requirements.txt
 EXAMPLES_REQUIREMENTS=(
   timm==0.6.13
-  torchaudio=="2.2.0.${NIGHTLY_VERSION}"
```

Comment:
This should be in main as well?

Reply:
thanks - updated https://github.com/pytorch/executorch/pull/4000

Reply:
note: merge when torch 2.4 pip package is released

Reply:
OK seems that we need to merge this on release day